src/main/java/github/weichware10/toolbox/codecharts/CodeChartsController.java

```diff
@@ -28,9 +41,35 @@ void endTest() {
         codeCharts.endTest();
     }
 
-    @FXML
-    void initialize() {
+    private void setImageViewWidth(ObservableDoubleValue width) {
+        codeChartsImageView.setFitWidth(width.get());
+        codeChartsGridPane.setPrefWidth(width.get());
+    }
+
+    private void setImageViewHeight(ObservableDoubleValue height) {
+        codeChartsImageView.setFitHeight(height.get());
+        codeChartsGridPane.setPrefHeight(height.get());
+    }
 
+    private void setGridPane(){
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>WhitespaceAround: '{' is not preceded with whitespace.